packages/loot-core/src/platform/server/fs/index.web.ts

```diff
@@ -77,15 +78,15 @@ async function _readFile(filepath, opts?: { encoding?: string }) {
     !filepath.endsWith('.sqlite')
   ) {
     if (!_exists(filepath)) {
-      throw new Error('File does not exist: ' + filepath);
+      throw new Error(t('File does not exist:') + ' ' + filepath);
```

Comment:
Same here, these aren't shown to the user so no need to translate them. Ideally they'd be english so we can read them in English when people come for support